src/ng-option-highlight/.eslintrc.json

```diff
@@ -0,0 +1,77 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": [
+    "!**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "src/ng-option-highlight/tsconfig.lib.json",
+          "src/ng-option-highlight/tsconfig.spec.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "rules": {
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "lib",
+            "style": "kebab-case"
+          }
+        ],
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "lib",
+            "style": "camelCase"
+          }
+        ],
+        "@typescript-eslint/dot-notation": "off",
+        "@typescript-eslint/explicit-member-accessibility": [
+          "off",
+          {
+            "accessibility": "explicit"
+          }
+        ],
+        "@typescript-eslint/member-delimiter-style": [
+          "off",
+          {
+            "multiline": {
+              "delimiter": "none",
+              "requireLast": true
+            },
+            "singleline": {
+              "delimiter": "semi",
+              "requireLast": false
+            }
+          }
+        ],
+        "@typescript-eslint/semi": [
+          "off",
+          null
+        ],
+        "brace-style": [
+          "error",
+          "1tbs"
+        ],
+        "id-blacklist": "off",
+        "id-match": "off",
+        "no-redeclare": "error",
+        "no-trailing-spaces": "off",
+        "no-underscore-dangle": "off"
```

Comment:
Should we fix these so we don't have to turn these off? I assume they must be triggering errors if they're not matching what was in the previous [tslist.json](https://github.com/ng-select/ng-select/pull/1996/files#diff-b09e073e9fc2668f089c105170728e74f159eccce9163ec6f979b6d182916d70)

Reply:
`@angular-eslint` library converted all rules from `tslint.json` to this file. But to be honest I haven't looked at the file closely, since they do good job of it. But I am not sure if we need to tackle this in this PR or have separate for it.